spinalcordtoolbox/process_seg.py

```diff
@@ -100,6 +100,20 @@ def compute_shape(segmentation, angle_correction=True, centerline_path=None, par
                                ncols=80):
         # Extract 2D patch
         current_patch = im_segr.data[:, :, iz]
```

Comment:
`current_patch` in PyCharm:

![image](https://github.com/user-attachments/assets/7b2f6598-ccd6-4b89-892d-8f43e5f8e936)